pkg/handlers/generic/lifecycle/addons/helmaddon.go

```diff
@@ -231,7 +231,7 @@ func waitToBeReady(
 				if obj.Generation != obj.Status.ObservedGeneration {
 					return false, nil
 				}
-				return conditions.IsTrue(obj, caaphv1.HelmReleaseProxiesReadyCondition), nil
+				return conditions.IsTrue(obj, clusterv1.ReadyCondition), nil
```

Comment:
`HelmReleaseProxiesReadyCondition` returns too early.

Reply:
Thanks! It's that `conditions.SetSummary` that sets the ready condition. This will possibly change when CAAPH supports v1beta2 conditions so we will need to look out for this.